onboard-app-guilherme/app/user/[id].tsx

```diff
@@ -0,0 +1,63 @@
+import { useQuery } from "@apollo/client/react";
+import { useLocalSearchParams, useRouter } from "expo-router";
+import { ActivityIndicator, Text, TouchableOpacity, View } from "react-native";
+import { GET_USER_QUERY } from "../../src/graphql/queries";
+import type { UserDetailsResponse, UserDetailsVariables } from "../../src/graphql/types";
+
+export default function UserDetailsScreen() {
+  const router = useRouter();
+  const { id } = useLocalSearchParams<{ id: string }>();
+
+  const { data, loading, error } = useQuery<UserDetailsResponse, UserDetailsVariables>(GET_USER_QUERY, {
+    variables: { id: String(id) },
```

Comment:
O id do local search params já vem como string, então não precisa converter aqui
```suggestion
    variables: { id },
```